setup.py

```diff
@@ -7,15 +7,20 @@
 from shutil import which
 from skbuild import setup
 
-# CMake Options
 CSP_USE_VCPKG = os.environ.get("CSP_USE_VCPKG", "1").lower() in ("1", "on")
+# Allow arg to override default / env
+if "--csp_no_vcpkg" in sys.argv:
+    CSP_USE_VCPKG = False
+    sys.argv.remove("--csp_no_vcpkg")
+
+# CMake Options
 CMAKE_OPTIONS = (
     ("CSP_BUILD_NO_CXX_ABI", "0"),
     ("CSP_BUILD_TESTS", "1"),
     ("CSP_MANYLINUX", "0"),
     ("CSP_BUILD_KAFKA_ADAPTER", "1"),
     ("CSP_BUILD_PARQUET_ADAPTER", "1"),
-    ("CSP_BUILD_WS_CLIENT_ADAPTER", "1"),
+    ("CSP_BUILD_WS_CLIENT_ADAPTER", str(int(platform.system() != "Windows"))),
```

Comment:
lets make sure to open an issue to build this down the road for windows.

Reply:
https://github.com/Point72/csp/issues/236